package.json

```diff
@@ -46,6 +46,7 @@
     "@bower_components/ng-ckeditor": "esvit/ng-ckeditor#^0.2.1",
     "@bower_components/ng-slide-down": "TheRusskiy/ng-slide-down#^1.0.0",
     "@bower_components/ovh-angular-browser-alert": "ovh-ux/ovh-angular-browser-alert#^1.0.0",
+    "@bower_components/angular-scroll-glue": "Luegg/angularjs-scroll-glue#^2.2.0",
```

Comment:
This dependency entry came from the `bower-away` migration I guess.

Prefer using:
```json
"angularjs-scroll-glue": "^2.2.0"
```